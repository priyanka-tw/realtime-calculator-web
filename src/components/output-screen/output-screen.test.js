import React from 'react';
import {render} from '@testing-library/react';
import 'jest-styled-components';
import OutputScreen from "./output-screen";

describe('output screen', () => {

    it('should render component', () => {
        const {container} = render(<OutputScreen value={"test-value"}/>);

        expect(container).toMatchSnapshot();
    });

});
