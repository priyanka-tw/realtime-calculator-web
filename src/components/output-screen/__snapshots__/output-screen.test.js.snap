// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`output screen should render component 1`] = `
.c0 {
  color: black;
  font-size: 16pt;
  font-family: verdana-bold;
  background-color: #A9A9A9;
  border: none;
  outline: none;
  text-align: right;
  width: 236px;
  height: 60px;
}

<div>
  <input
    class="c0"
    data-testid="output-screen"
    readonly=""
    value="test-value"
  />
</div>
`;
